drivers/flash/flash_stm32_xspi.c

```diff
@@ -0,0 +1,2076 @@
+/*
+ * Copyright (c) 2022-2024 STMicroelectronics
+ * Copyright (c) 2022 Georgij Cernysiov
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * **************************************************************************
+ * xSPI flash controller driver for stm32 serie with xSPI periherals
+ * This driver is based on the stm32Cube HAL XSPI driver
+ * with one xspi DTS NODE
+ * **************************************************************************
+ */
+#define DT_DRV_COMPAT st_stm32_xspi_nor
+
+#include <errno.h>
+#include <zephyr/kernel.h>
+#include <zephyr/toolchain.h>
+#include <zephyr/arch/common/ffs.h>
+#include <zephyr/sys/util.h>
```

Comment:
Are these headers needed?

Reply:
includes are not needed, REMOVED
(also not needed in other flash_stm32_ospi.c/flash_stm32_qspi.c files)